CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportSessionData.java

```diff
@@ -73,7 +74,13 @@ public boolean storeSessionError(final ErrorReportEntry entry) {
             final String comparisonData = sb.toString(); 
             final int errCount = sessionErrors.size();
             for (int i = 0; i < errCount; i++) {
-                if (sessionErrors.get(i).getErrorLevel().equals(entry.getErrorLevel()) && sessionErrors.get(i).getErrorData().startsWith(comparisonData)) {
+                boolean allConstyLinesMatch = true;
+                for (final String constyLine : constyLines) {
+                    if (!sessionErrors.get(i).getErrorData().contains(constyLine)) {
+                        allConstyLinesMatch = false;
+                    }
```

Comment:
might be worth adding a break here. The loop is only determining whether `allConstyLinesMatch` should become false and once it does, it then stays that way for the remainder of the for loop so the remaining iterations are redundant after the if condition if fulfilled (and so might as well exit the loop at that stage as well).